internal/pkg/config/v1/config.go

```diff
@@ -271,6 +248,36 @@ func (c *Config) Save() error {
 	return nil
 }
 
+func (c *Config) encryptContextStateTokens(tempAuthToken, tempAuthRefreshToken string) error {
+	if c.Context().State.Salt == nil || c.Context().State.Nonce == nil {
```

Comment:
`GetSalt()` might also be helpful in preventing a `panic()` here.
```suggestion
	if c.GetState().Salt == nil || c.GetState().Nonce == nil {
```